p2p/host.go

```diff
@@ -286,6 +279,33 @@ func NewHost(cfg HostConfig) (Host, error) {
 	return h, nil
 }
 
+// connectionManager creates a new connection manager and configures libp2p to use the
+// given connection manager.
+// lo and hi are watermarks governing the number of connections that'll be maintained.
+// When the peer count exceeds the 'high watermark', as many peers will be pruned (and
+// their connections terminated) until 'low watermark' peers remain.
+func connectionManager(low int, high int) libp2p_config.Option {
+	if low > 0 && high > low {
```

Comment:
Isn't high and low already checked above ?

Reply:
We're looking to see if high>low at the top. but here we are making sure they are set to be used by connection manager; otherwise, we don't initiate connection manager. In this way, using Connection Manager will be an optional feature. If we don't want to use it, just set both high and low to zero.